modules/timeutil/timestamp.go

```diff
@@ -58,6 +58,16 @@ func (ts TimeStamp) AsTime() (tm time.Time) {
 	return ts.AsTimeInLocation(setting.DefaultUILocation)
 }
 
+// AsOptionalTime convert timestamp as *time.Time in Local locale. Returns nil if the time is zero.
+func (ts TimeStamp) AsOptionalTime() (tm *time.Time) {
+	if ts.IsZero() {
+		return nil
+	} else {
+		timePointer := ts.AsTimeInLocation(setting.DefaultUILocation)
```

Comment:
Does this convert timezone? I would not, we should just store UTC in the database and convert for presentation only.

Reply:
It is used in the API, so I think it should use UTC

Reply:
Yeah we should prefer UTC on both db and api. Ideally we should convert timezone in JS but that is still a TODO, so current presentation convertion happens in the backend IIRC.

Reply:
This is the soucre of `TimeStampNow()`:
https://github.com/go-gitea/gitea/blob/40ba750c4bf1f3f5f8dff5af57b2db4b600f237f/modules/timeutil/timestamp.go#L34-L39
It is a internal function of Gitea. If it not uses UTC, it might affect other parts of Gitea too.

Reply:
fine for now.